omnigibson/objects/primitive_object.py

```diff
@@ -347,6 +346,7 @@ def _create_prim_with_same_kwargs(self, prim_path, name, load_config):
     def _dump_state(self):
         state = super()._dump_state()
         # state["extents"] = self._extents
+        # TODO: Why are these here and not in the init kwargs?
```

Comment:
maybe because the user might change these values during simulation, and dump/load_state should use the new value?

Reply:
@cgokmen

Reply:
I think I might have implemented this -- and yes, that's because the extents explicitly get updated when these values get updated IIRC

Reply:
OK sg pleas eundo

Reply:
TODO: remove the TODO